files/bootstrap.sh

```diff
@@ -14,7 +14,7 @@ if [[ -e $HOME/pypy-$PYPY_VERSION-linux64.tar.bz2 ]]; then
   tar -xjf $HOME/pypy-$PYPY_VERSION-linux64.tar.bz2
   rm -rf $HOME/pypy-$PYPY_VERSION-linux64.tar.bz2
 else
-  wget -O - https://bitbucket.org/pypy/pypy/downloads/pypy-$PYPY_VERSION-linux64.tar.bz2 |tar -xjf -
+  wget --no-check-certificate -O - https://bitbucket.org/pypy/pypy/downloads/pypy-$PYPY_VERSION-linux64.tar.bz2 |tar -xjf -
```

Comment:
Seeing this included would cause me to work on a  pull request that removes it. 

Better automate installation of the required certificate than throwing what protection https gives out of the window.

But even without this change, it works (on my machine).